lld/COFF/Options.td

```diff
@@ -132,6 +132,8 @@ def manifestinput : P<
 // a reserved keyword in tablegen.
 def incl : Joined<["/", "-", "/?", "-?"], "include:">,
     HelpText<"Force symbol to be added to symbol table as undefined one">;
+def incl_glob : Joined<["/", "-", "/?", "-?"], "includeglob:">,
```

Comment:
I think it would be better to place this new option in the separate section below, among other LLD extensions.

Reply:
Yeah that's totally fair. I've moved it to the appropriate `// LLD extensions` section.